gamedata/tf2.cattr_starterpack.txt

```diff
@@ -679,20 +679,21 @@
 				"signature"			"CTFGameRules::ApplyOnDamageAliveModifyRules()"
 				"linux"
 				{
-					"offset"		"1CD8h"
-					
+					// 95% sure about this
+					"offset"		"106Eh"
+
 					// mulss xmm2, *float == 0.5
-					"verify"		"\xF3\x0F\x59\x15\x2A\x2A\x2A\x2A"
+					"verify"		"\xF3\x0F\x59\x15\x6C\x2A\x2A\x2A"
```

Comment:
The `\x6C` is part of the address to the float, which may break after any update; the original verify bytesig is fine in this case.  Same with the Windows one.

Reply:
Got it. I must have overlooked it then, since I changed the last three bytes but not that one.